sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -15,9 +15,11 @@
 import json
 import logging
 
+import avalon_crypto_utils.crypto.crypto as crypto
```

Comment:
this library has been removed with the latest openssl change, so will need rework

Reply:
Done